envelope.md

```diff
@@ -1,8 +1,8 @@
 # DSSE Envelope
 
-March 03, 2021
+May 10, 2024
 
-Version 1.0.0
+Version 1.0.1
```

Comment:
This is to account for a patch update that was not officially observed until now?

Reply:
yes, we've had clarifications in the spec since v1.0.0 on `master` that was never tagged.